apps/gidd/views.py

```diff
@@ -606,6 +615,282 @@ def export(self, request):
         return response
 
 
+class DisaggregationViewSet(ListOnlyViewSetMixin):
+    serializer_class = DisaggregationSerializer
+    filter_backends = (DjangoFilterBackend, filters.OrderingFilter, filters.SearchFilter)
+    filterset_class = DisaggregationFilterst
+
+    def get_queryset(self):
+        track_gidd(
+            self.request.GET.get('client_id'),
+            ExternalApiDump.ExternalApiType.GIDD_DISAGGREGATION_REST,
+            viewset=self,
+        )
+        return GiddFigure.objects.all()
+
+    def _get_term(self, term):
+        return Figure.FIGURE_TERMS.get(term).name if term else None
+
+    def _get_category(self, category):
+        return Figure.FIGURE_CATEGORY_TYPES.get(category).name if category else None
+
+    def _get_cause(self, cause):
+        return Crisis.CRISIS_TYPE.get(cause).name if cause else None
+
+    def _get_displacement_occurred(self, displacement_occurred):
+        return Figure.DISPLACEMENT_OCCURRED.get(displacement_occurred).name if displacement_occurred else None
+
+    def export_disaggregated_geojson(self, qs):
+
+        def format_coordinate(coordinate: str) -> typing.Tuple[float, float]:
+            lat, lng = coordinate.split(', ')
+            return (float(lng), float(lat))
+
+        def format_coordinates(coordinates: typing.List[str]):
+            return [format_coordinate(x) for x in coordinates]
+
+        qs = qs.filter(
+            locations_coordinates__isnull=False,
+        ).annotate(
+            event_main_trigger=Case(
+                When(
+                    gidd_event__cause=Crisis.CRISIS_TYPE.CONFLICT,
+                    then=F('gidd_event__violence_sub_type__name')
```

Comment:
```suggestion
                    then=F('gidd_event__violence_type__name')
```

@matemies suggested that we use violence type instead of violence sub type for external purpose.